### ISSUE TITLE
Missing Twitter Account Label from User Settings

### ISSUE THREAD

**Issue by boubaker** [Meeds-io/meeds#2714]
**Steps to reproduce**

1. Go to User settings
2. See attached image

![image](https://github.com/user-attachments/assets/97b489f1-8057-47eb-88e8-c500b0fa046f)

**Comment by srenault-meeds** [comment]
Yes thanks. Already identified here: https://github.com/Meeds-io/meeds/issues/2692
